### ISSUE TITLE
[UWP][Accessibility] SelectActions should be exposed to accessibility as links

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#3591]
Currently, elements with a `selectAction` are shown to accessibility as `button`, but they should be of `link` type.
